### ISSUE TITLE
这里似乎有错误？

### ISSUE THREAD

**Issue by AAAceerr** [machengcheng2016/CPE-LTSSL#4]
作者你好，关于cpe.py
最后的验证部分，源代码是传入‘logits_2’
def evaluate(self, eval_dest='eval', out_key='logits', return_logits=False):
        return super().evaluate(eval_dest=eval_dest, out_key='logits_2', return_logits=return_logits)
但是在前文中
logits2_all = self.classifier2(feat)
        c = logits2_all.shape[0] // 3
        logitsH2 = logits2_all[:c,:]
        logitsM2 = logits2_all[c:c*2,:]
        logitsT2 = logits2_all[c*2:,:]
        logits_2 = (logitsH2 + logitsM2 + logitsT2) / 3
        
        results_dict['logitsH2'] = logitsH2
        results_dict['logitsM2'] = logitsM2
        results_dict['logitsT2'] = logitsT2
        results_dict['logits_2'] = logits_2

这是一个很明显的错误，logits_2的batchsize只有1/3个batchsize，应该传入的是logits2_all

还是我理解错误了呢？按照源代码确实出现了维度不匹配，只有1/3的batchsize。

等待回复
